common/lib/xmodule/xmodule/modulestore/tests/persistent_factories.py

```diff
@@ -1,99 +0,0 @@
-"""Provides factories for Split."""
```

Comment:
This file had been removed on master (I removed it...) And it was causing pylint errors.